engine/engine/content/builtins/render/default.render_script

```diff
@@ -67,8 +67,10 @@ local function update_clear_color(state, color)
 end
 
 local function update_camera(camera, state)
-    camera.proj = camera.projection_fn(camera, state)
-    camera.frustum.frustum = camera.proj * camera.view
```

Comment:
renamed the inner table from frustum to options, since that is how it's used later in the rendering (i.e as the second parameter to draw, which is a table of options). This was a bit confusing for users, so this should make it more clear.